app-arch/ancient/ancient-2.0.0.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="Modern decompressor for old data compression formats"
+DESCRIPTION="This is a collection of decompression routines for old formats \
+popular in the Amiga, Atari computers and some other systems from 80's \
+and 90's as well as some that are currently used which were used in a some \
+specific way in these old systems."
+HOMEPAGE="https://github.com/temisu/ancient"
+COPYRIGHT="2017-2022, Teemu Suutari and contributors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/temisu/ancient/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="9c6b31e4d968ec3adb25596a0b30405d1080d3de0546f1485ecb0ac2eca6261d"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="2"
```

Comment:
In this case libVersion is the same as portVersion (2.0.0), so:
```diff
-libVersion="2"
+libVersion="$portVersion"
```

Reply:
SO files only have "2" in them

Reply:
If you look in the devel package you'll see `libancient.so.2`, `libancient.so.2.0.0` and `libancient.so` in there.

Reply:
```
/boot/system/lib/x86> tree | grep libancient
├── libancient.so.2 -> libancient.so.2.0.0
├── libancient.so.2.0.0
```

Reply:
I just tested again something compiled against libancient, and did a lddtree, and libancient.so.2 is the only file name the resulting binary searches for.

But if is preferred to rather use 2.0.0 the provides, I can update recipe later when I return from work trip

Reply:
As you can see in the above tree search, `libancient.so.2` is linked against `libancient.so.2.0.0`

Reply:
This has now been updated